layout/_macro/post.swig

```diff
@@ -78,11 +78,18 @@
             #}</time>
             {% endif %}
 
-            {% if theme.post_meta.created_at and theme.post_meta.updated_at %}
-              <span class="post-meta-divider">|</span>
+            {% set date_updated_diff = date(post.date, config_date_format) != date(post.updated, config.date_format) %}
+            {% if theme.post_meta.updated_at %}
+              {% if !theme.post_meta.updated_diff || theme.post_meta.updated_diff &&  date_updated_diff %}
```

Comment:
One exess space before `date_updated_diff` string here.

Reply:
Fixed the whitespace, and everything was tested.